portfolio/src/main/webapp/gallery-script.js

```diff
@@ -18,24 +16,42 @@ const addProjectsToDOM = async (json) => {
     json will always contain a top-level object which has a single member.
     this member will be an array of objects. these objects will have three properties:
     - title : a string containing the title of the project
-    - URL : a string which contains a URL linking to the blog post about
+    - URL : a string which contains a URL linking to the source code of
         the project
     - preview : a string which contains a URL linking to the preview image
         for this project
+    - description : a string which contains a short description of this project
+    - blog_link : a string which contains a URL linking to a blog post about
+        the project
   */
+  console.log(json);
```

Comment:
Is this meant to be there?

Reply:
it gets removed in a future commit on the next pr 👍 https://github.com/thomasogara/step-repository/pull/5/commits/85588e1a2e98cda534771165e522b3e2c8ff1cc1